tests/manual/performance/editorinit.html

```diff
@@ -0,0 +1,17 @@
+<div id="test-controls">
+	Initialize editor with:
+	<button id="small-content" data-file-name="small" disabled>small</button>
+	<button id="medium-content" data-file-name="medium" disabled>medium</button>
+	<button id="large-content" data-file-name="large" disabled>large</button>
+	HTML payload.
+</div>
+
+<style>
+	.editor {
+		display: none;
+	}
+</style>
+
+<div id="editor_small" class="editor"></div>
```

Comment:
The `<hr>` would be nice here as in other manual tests (small issue but would be nice to have same visual pattern for those tests).